### ISSUE TITLE
Php 7.1 doesn't work with 1.11 and 1.12 versions (at least)

### ISSUE THREAD

**Issue by igronus** [PHPOffice/PhpSpreadsheet#3996]
This is:

```
- [x] a bug report
- [ ] a feature request
- [x] **not** a usage question
```

### What is the expected behavior?

The library versions 1.11 and 1.12 should work with PHP 7.1 according to composer.json requirements.

### What is the current behavior?

The library works with PHP 7.1 if the version is 1.6 for example. But 1.11 and 1.12 versions do not.

### What are the steps to reproduce?

This code causes infinite loading and memory consumption.

```php
$objPHPExcel = IOFactory::load($templatePath);
```

### What features do you think are causing the issue

- [x] Reader
- [ ] Writer
- [ ] Styles
- [ ] Data Validations
- [ ] Formula Calculations
- [ ] Charts
- [ ] AutoFilter
- [ ] Form Elements

At least the reader is affected, but can't say for sure, probably something happening on the project level while adding newer php versions or while dropping older ones.

### Does an issue affect all spreadsheet file formats? If not, which formats are affected?

No.

### Which versions of PhpSpreadsheet and PHP are affected?

1.11 and 1.12 at least. Maybe some others higher than 1.6 and below 1.11.

**Comment by oleibman** [comment]
I will just get this out of the way up front - Php 7.1 became unsupported over 4 years ago, and you are running considerable risk if you continue to use it. Likewise PhpSpreadsheet 1.12 was replaced almost 4 years ago, and new versions have been tagged 23 times since then.

When Php 1.12 was released, it worked with Php 7.1, and should still do so. That doesn't mean it didn't have bugs. We receive bug reports and fix them all the time. We do not, however, retrofit the bug fixes to earlier releases. So, I guess a good first step for you would be to see how your spreadsheet works with current releases of PhpSpreadsheet and Php. If it works, you know what needs to be done. If it doesn't work, please file a bug with sufficient info (e.g. failing spreadsheet) to allow us to investigate.

**Comment by oleibman** [comment]
No update in 4 months. Closing.
